docs/runtime.ios.md

```diff
@@ -1,74 +1,72 @@
-## ~`getRuntimeConfiguration()`~
-
-on Apple devices, this data is retrieved from the following string-replacements
-
-- [BrowserServices Kit String replacements](https://github.com/duckduckgo/BrowserServicesKit/blob/main/Sources/BrowserServicesKit/Autofill/AutofillUserScript+SourceProvider.swift#L54-L56)
+## String replacement
+On iOS and macOS, we string-replace these variables:
```

Comment:
Not long ago we also moved Apple platforms to using `getRuntimeConfiguration` instead of string replacement.